formatting the new function by Andre and adding documentation

lubianat opened the pull request.

lubianat committed:
formatting the new function by Andre and adding documentation

lubianat commented:
This approving review system is awesome! But I wonder if it will work...

pedrostrusso reviewed (commented):
I would suggest returning the `return_index` to `value` (and, consequently, removing the `!`) as in the original syntax so it would be consistent with `grep` (thus lessening the burden on users).

lecardozo commented on R/search_multiple_names.R:
```diff
@@ -0,0 +1,25 @@
+#' search_multiple_names(vector_of_interest, query_vector = query, return_index = TRUE)
+
+
+search_multiple_names <- function(vector_of_interest, query_vector, perl = TRUE, return_index = FALSE) {
```
Why not just pass ellipsis and pipe it directly into `grep` so the behaviour is whatever the user wants?
```suggestion
search_multiple_names <- function(vector_of_interest, query_vector, ...) {
```

nicolau committed:
Update R/search_multiple_names.R

Co-Authored-By: Lucas Cardozo <lucasecardozo@gmail.com>

nicolau committed:
Adding possibility to search by OR or AND

Adding a parameter logical_operator to define logical operator OR or AND to search.

nicolau committed:
logical  operator was removed and fixed examples

nicolau committed:
Add ellipsis in the grep function
